mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -244,14 +247,13 @@ impl mayastor_server::Mayastor for MayastorSvc {
         request: Request<DestroyReplicaRequest>,
     ) -> GrpcResult<Null> {
         let args = request.into_inner();
-        let rx = rpc_submit(async move {
-            match Bdev::lookup_by_name(&args.uuid) {
-                Some(b) => {
-                    let lvol = Lvol::try_from(b)?;
-                    lvol.destroy().await.map(|_r| Null {})
-                }
-                None => Ok(Null {}),
+        let rx = rpc_submit::<_, _, LvsError>(async move {
+            if let Some(bdev) = Bdev::lookup_by_name(&args.uuid) {
+                let lvol = Lvol::try_from(bdev)?;
+                lvol.destroy().await?;
+                sync_config().await;
```

Comment:
Again, consider what context you are running this in. We are basically doing file IO within the context of SPDK, and its not needed to store this information here.

Reply:
removed